### PR TITLE
GUI: add wrappers for event counts (cospeciation, duplication, transfer, and loss)

### PR DESCRIPTION
**Pull request by Qing-Yang-1211**
This is the tentative version of the gui for the user study.
Added the wrappers for event counts (cospeciation, duplication, transfer, and loss). This doesn't include the "View frequency" and "View internal node names" functions in the view reconciliations-one MPR tkinter window and those will be added later after they work correctly (pull request #126). 

I have tested the event counts wrappers by loading in different files and changing dtl costs to compute new reconciliations. I could see that the numbers were all changing and the number of MPRs looked correct but I couldn't know the other numbers.

### CODE REVIEW

File: empress_gui.py [hunk 1]
```diff
@@ -150,11 +150,15 @@ def init_compute_reconciliations(self):
         self.compute_reconciliations_btn.grid(row=3, column=0)
 
         self.recon_MPRs_label = tk.Label(self.recon_nums_frame)
-        self.num_MPRs_label = tk.Label(self.recon_nums_frame)
+        self.MPRs_count_label = tk.Label(self.recon_nums_frame)
```
Comment (ssantichaivekin) [inline empress_gui.py:153]: Lower case variable names :)

File: empress_gui.py [hunk 2]
```diff
@@ -263,26 +269,33 @@ def reset(self):
         self.loss_entry_box.destroy()
 
         self.recon_MPRs_label.destroy()
-        self.num_MPRs_label.destroy()
+        self.MPRs_count_label.destroy()
```
Comment (ssantichaivekin) [inline empress_gui.py:272]: Lower case variable names :)

File: empress_gui.py [hunk 3]
```diff
@@ -544,27 +557,48 @@ def update_compute_reconciliations_btn(self):
...
-        self.num_MPRs = App.recon_graph.n_recon
+        self.MPRs_count = App.recon_graph.n_recon
+        self.cospec_count, self.dup_count, self.trans_count, self.loss_count = App.recon_graph.median().count_events()
         if not self.recon_info_displayed:
             # Display numeric reconciliation results
             self.recon_MPRs_label = tk.Label(self.recon_nums_frame, text="Number of MPRs: ")
             self.recon_MPRs_label.grid(row=0, column=0, sticky="w")
-            self.num_MPRs_label = tk.Label(self.recon_nums_frame, text=self.num_MPRs)
-            self.num_MPRs_label.grid(row=0, column=1, sticky="w")
+            self.MPRs_count_label = tk.Label(self.recon_nums_frame, text=self.MPRs_count)
```
Comment (ssantichaivekin) [inline empress_gui.py:566]: Lower case variable names :)

File: empress_gui.py [hunk 4]
```diff
@@ -624,7 +658,7 @@ def validate_num_cluster_input(self, input_after_change: str):
         """The number of clusters is only allowed to be an integer that >= 1 and <= the number of MPRs."""
         try:
             val = int(input_after_change)
-            if val >= 1 and val <= self.num_MPRs:
+            if val >= 1 and val <= self.MPRs_count:
```
Comment (ssantichaivekin) [inline empress_gui.py:661]: Lower case variable names :)

File: empress_gui.py (hunk 1 above)
Comment (Qing-Yang-1211) [reply empress_gui.py:153]: Done.

File: empress_gui.py (hunk 2 above)
Comment (Qing-Yang-1211) [reply empress_gui.py:272]: Done.

File: empress_gui.py (hunk 3 above)
Comment (Qing-Yang-1211) [reply empress_gui.py:566]: Done.

File: empress_gui.py (hunk 4 above)
Comment (Qing-Yang-1211) [reply empress_gui.py:661]: Done.

**Pull request merged**